src/controller/CHIPCommissionableNodeController.h

```diff
@@ -54,6 +55,10 @@ class DLL_EXPORT CommissionableNodeController : public AbstractMdnsDiscoveryCont
         ChipLogError(Controller, "Unsupported operation CommissionableNodeController::OnNodeIdResolutionFailed");
     }
 
+    CHIP_ERROR RequestCommissioning(int commissionerIdx);
```

Comment:
Document, please.  This does not have trivial semantics in terms of what the argument means.